### ISSUE TITLE
USB stops working

### ISSUE THREAD

**Issue by tostc** [ValveSoftware/steam-for-linux#8322]
#### Your system information

ASRock 970A-G/3.1
AMD FX(tm)-6300 Six-Core Processor
NVIDIA Corporation NVIDIA GeForce GTX 750 Ti/PCIe/SSE2 (Proprietary driver)
16 GB Ram

* Steam client version (build number or date): 16. Dec. 2021
* Distribution (e.g. Ubuntu): Manjaro Linux 64 Bit (Kernel: 5.14.21-2-MANJARO)
* Opted into Steam client beta?: No
* Have you checked for system updates?: Yes

#### Please describe your issue in as much detail as possible:

When Steam is only running in the background, my entire system sound usually cracks. This occurs on both Steam and non-Steam games. All my USB devices stop working for a second. So when I write something, some letters do not appear on the screen, and sometimes my mouse clicks are not recognized either. As soon as I close steam completely, the problems are gone. Sometimes my entire USB driver crashes so I have to restart my computer to get it working again.

#### Steps for reproducing this issue:

1. Start steam
2. Go to your browser and type something or just listen to your music or play any game
3. Alternative hover over cards in the steam shop or your library and listen music the sound will crackling

**Comment by kisak-valve** [comment]
Hello @tostc, this reads like a kernel issue that should be reported to the maintainer of the USB subsystem of the Linux kernel. The fact that Steam agitates an underlying kernel or hardware issue is coincidental.

**Comment by tostc** [comment]
Thanks for the answer. I will report the problem to the kernel USB subsystem maintainers.

**Comment by tostc** [comment]
I reported the problem to the kernel team and, with the help of the USB maintainer, found out that the proprietary nvidia driver was causing the problem. I think this is not a Steam issue so I will close this issue and contact nvidia about this issue.

**Comment by tostc** [comment]
Hey it's me again This is just a quick update I'd like to share so that other users with the same problem can fix it in the future too. I disabled some options in the Steam settings and now my problems are gone. 
1. Under "Interface" I turned off "GPU accelerated rendering in web views"
2. I have activated the option "Low Performance Mode" under "Library"
Now the audio crackling and usb issues are gone.
